.github/workflows/appengine-deploy-it.yml

```diff
@@ -0,0 +1,44 @@
+name: appengine-deploy Integration
+on: [push, pull_request]
+jobs:
+  gcloud:
+    name: with setup-gcloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./setup-gcloud
+      with:
+        service_account_email: ${{ secrets.APPENGINE_DEPLOY_SA_EMAIL }}
```

Comment:
@craigdbarber Can you help me set up secrets for this repo? Do we have a test project?

Reply:
@averikitsch happy to :)
Will sync offline.